src/fileformat/file_format/pe/authenticode/authenticode_structs.h

```diff
@@ -0,0 +1,91 @@
+/* Author Marek Milkovič - metthal */
```

Comment:
Again, proper header. If you want to keep the author, you can do so in the Doxygen format. This is not standard in the other sources, but in this case @metthal can be mentioned.